base/ca/src/main/java/org/dogtagpki/server/ca/rest/ProfileService.java

```diff
@@ -110,7 +110,7 @@ public ProfileService() {
     }
 
     @Override
-    public Response listProfiles(Integer start, Integer size) {
+    public Response listProfiles(Integer start, Integer size, Boolean visible, Boolean enable) {
```

Comment:
Should we add a filter for `enableBy` too? It's not required and it doesn't have to be done in this PR.

Reply:
Done